local/bin/py/build/actions/integrations.py

```diff
@@ -812,6 +812,9 @@ def process_integration_readme(self, file_name, marketplace=False):
                 # Remove % delimiters which can cause Hugo's to break with nested shortcodes.
                 result = result.replace("{{%", "{{<").replace("%}}", ">}}")
 
+                # Preserve % delimiters for markdown shortcodes.
+                result = result.replace("{{< aws-permissions >}}", "{{% aws-permissions %}}")
```

Comment:
This is a quick fix, we will need a better long-term solution ensuring markdown shortcodes are handled properly when pulling content into the docs build.